libraries/AP_AccelCal/AP_AccelCal.cpp

```diff
@@ -110,7 +117,9 @@ void AP_AccelCal::update()
                 uint32_t now = AP_HAL::millis();
                 if (now - _last_position_request_ms > AP_ACCELCAL_POSITION_REQUEST_INTERVAL_MS) {
                     _last_position_request_ms = now;
+#if !defined(HAL_BUILD_AP_PERIPH)
```

Comment:
```suggestion
`#ifndef HAL_NO_GCS  || !HAL_NO_GCS`
```

I think that's the pattern required.

It should be
```suggestion
#if !HAL_NO_GCS
```

but I'm not sure it is always defined ATM.  And it really should be `HAL_GCS_ENABLED`

In any case, we should trust the define.

Similarly elsewhere in here.